governance/polls/Increase the ilk.hole Parameter for Several Collaterals - November 15, 2021.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Increase the ilk.hole Parameter for Several Collaterals - November 15, 2021
+summary: Signal your support or opposition for increasing ilk.hole Parameter for ETH-A, ETH-B, ETH-C, WBTC-A and WSTETH-A.
+discussion_link: https://forum.makerdao.com/t/auction-throughput-parameters-adjustments-nov-9-2021/11531
+vote_type: Plurality Voting
+categories:
+   - Risk Variable (Rates?*)
+   - Auctions
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-15T16:00:00
+end_date: 2021-11-18T16:00:00
+---
+# Poll: Increase the ilk.hole Parameter for Several Collaterals - November 15, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, November 15 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review 
+
+MKR Token Holders may vote in this poll to express support or opposition to increasing [ilk.hole](https://makerdao.world/en/learn/governance/param-local-liquidation-limit/) for the following vaults as follows:
+   * ETH-A: from current 40M to 65M.
+   * ETH-B: from current 25M to 30M.
+   * ETH-C: from current 30M to 35M.
+   * WBTC-A: from current 25M to 40M
+   * WSTETH-A: from current 3M to 7M
```

Comment:
As before, would spell out "million" and indicate that this is a figure in DAI

See this poll for an example of how you might format this section: https://github.com/makerdao/community/blob/master/governance/polls/Adjusting%20Auction%20Parameters%20for%20ETH-A%2C%20ETH-C%2C%20and%20WBTC-A%20-%20September%2020%2C%202021.md